engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -23,15 +23,25 @@ import pl.touk.nussknacker.engine.api.CirceUtil._
   */
 @ConfiguredJsonCodec sealed trait ParameterValidator {
 
-  def isValid(paramName: String, value: String, label: Option[String])(implicit nodeId: NodeId): Validated[PartSubGraphCompilationError, Unit]
+  val priority: Long = 0
+  protected val skipIfBlank: Boolean
 
+  final def isValid(paramName: String, value: String, label: Option[String])(implicit nodeId: NodeId): Validated[PartSubGraphCompilationError, Unit] = {
```

Comment:
I think isValid should stay as before.. We can add validate method where we will check this one if..  But this is only my opinion ;)

Reply:
But then everyone would have to remember to use this validate method for consistency, right? I don't know either, I was thinking this way would be clearer, but I may be wrong

Reply:
Ah, you meant to check it somewhere "higher"... Yeap, we could do it, but then we'd also have to rewrite tests testing the validators logic. I think it's better if the logic stays in one place.

Reply:
Search Results
Translation result
English
Polish
Let's look for this case:
I want test one validator, for example Max Value. When I put there empty string then a I will get true.. Is it really true? I don't thinks so ;)

Each validator should reuturn good result alsow for empty string value.. 

Now we have the same trick with checking emptyString but we moved it to some abstract place.. 

WDYF?